openmetadata-ui/src/main/resources/ui/src/pages/TestSuitePage/TestSuitePage.tsx

```diff
@@ -112,28 +114,39 @@ const TestSuitePage = () => {
   }, []);
 
   return (
-    <Row className="tw-w-full">
-      <Col span={24}>
-        <Table
-          columns={columns}
-          dataSource={testSuites.map((test) => ({ ...test, key: test.name }))}
-          loading={isLoading}
-          pagination={false}
-          size="small"
-        />
-      </Col>
-      {testSuites.length > PAGE_SIZE_MEDIUM && (
+    <PageLayoutV1>
+      <TitleBreadcrumb
+        titleLinks={[
+          {
+            name: 'Test Suites',
+            url: '',
+            activeTitle: true,
+          },
+        ]}
+      />
+      <Row className="w-full mt-4">
         <Col span={24}>
-          <NextPrevious
-            currentPage={testSuitePage}
-            pageSize={PAGE_SIZE_MEDIUM}
-            paging={testSuitePaging}
-            pagingHandler={testSuitePagingHandler}
-            totalCount={testSuitePaging.total}
+          <Table
+            columns={columns}
+            dataSource={testSuites.map((test) => ({ ...test, key: test.name }))}
+            loading={isLoading}
```

Comment:
```suggestion
          loading={{ spinning: loading, indicator: <Loader /> }}
```

This will override AntD default Loader with our